### PR TITLE
Added discord link

### PR DESCRIPTION
**Pull request by cyberwizard1001**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -132,6 +132,12 @@
       <a
         target="_blank"
         href="https://www.instagram.com/tedxamritavishwavidyapeetham/"
+      >
+        Discord
+      </a>
+      <a
+        target="_blank"
+        href="https://discord.gg/3DGrXvPa"
```
Comment (radiantly) [inline index.html]: This invite expires in a week - could this be replaced with a non-expiring one? Thanks!

**Review by radiantly — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (cyberwizard1001) [reply index.html]: Ah. One minute

File: index.html [hunk 2]
```diff
@@ -132,6 +132,12 @@
       <a
         target="_blank"
         href="https://www.instagram.com/tedxamritavishwavidyapeetham/"
+      >
+        Discord
+      </a>
+      <a
+        target="_blank"
+        href="https://discord.gg/sjVYcaGJZd"
```
Comment (cyberwizard1001) [inline index.html]: Link should be okay now

Comment (radiantly) [inline index.html]: Wrong server.

**Review by radiantly — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (cyberwizard1001) [reply index.html]: Sheet.

**Review by radiantly — APPROVED** [review]

**Review by radiantly — CHANGES_REQUESTED** [review]
The Instagram link leads to Discord and the Discord one leads to Instagram :P

**Review by radiantly — APPROVED** [review]
Awesome, thanks!

**Pull request merged**